certbot/log.py

```diff
@@ -359,10 +359,11 @@ def post_arg_parse_except_hook(exc_type, exc_value, trace, debug, log_path):
         if issubclass(exc_type, errors.Error):
             sys.exit(exc_value)
         print('An unexpected error occurred:', file=sys.stderr)
-        if messages.is_acme_error(exc_value):
-            # Remove the ACME error prefix from the exception
-            _, _, exc_str = str(exc_value).partition(':: ')
-            print(exc_str, file=sys.stderr)
+        if isinstance(exc_value, messages.Error):
+            if (exc_value.typ is not None) and messages.is_acme_error(exc_value):
```

Comment:
So I feel bad because I suggested that we do things this way, but I think it'd be good to put the `isinstance(exc_value, messages.Error)` and `exc_value.typ is not None` checks in `is_acme_error()` rather than here. There are other places that `is_acme_error()` is called and rather than making sure the caller remembers to do these checks, we can just do them in `is_acme_error()` so everyone benefits.